paper.md

```diff
@@ -1,32 +1,155 @@
 ---
-title: 'sourmash: a library for MinHash sketching of DNA'
+title: 'sourmash: a tool to quickly search, compare, and analyze genomic and metagenomic data sets'
 tags:
+  - FracMinHash
   - MinHash
   - k-mers
   - Python
+  - Rust
 authors:
- - name: C. Titus Brown
-   orcid: 0000-0001-6001-2677
-   affiliation: University of California, Davis
  - name: Luiz Irber
    orcid: 0000-0003-4371-9659
-   affiliation: University of California, Davis
-date: 13 Sep 2016
+   equal-contrib: true
+   affiliation: 1
+ - name: N. Tessa Pierce-Ward
+   orcid: 0000-0002-2942-5331
+   equal-contrib: true
+   affiliation: 1
+ - name: Mohamed Abuelanin
+   orcid: 0000-0002-3419-4785
+   affiliation: 1
+ - name: Harriet Alexander
+   orcid: 0000-0003-1308-8008
+   affiliation: 2
+ - name: Abhishek Anant
+   orcid:  0000-0002-5751-2010
+   affiliation: 9
+ - name: Keya Barve
+   orcid: 0000-0003-3241-2117
+   affiliation: 1
+ - name: Colton Baumler
+   orcid: 0000-0002-5926-7792
+   affiliation: 1
+ - name: Olga Botvinnik
+   orcid: 0000-0003-4412-7970
+   affiliation: 3
+ - name: Phillip Brooks
+   orcid: 0000-0003-3987-244X
+   affiliation: 1
+ - name: Daniel Dsouza
+   orcid: 0000-0001-7843-8596
+   affiliation: 9
+ - name: Laurent Gautier
+   orcid: 0000-0003-0638-3391
+   affiliation: 9
+ - name: Mahmudur Rahman Hera
+   orcid: 0000-0002-5992-9012
+   affiliation: 4
+ - name: Hannah Eve Houts
+   orcid: 0000-0002-7954-4793
+   affiliation: 1
+ - name: Lisa K. Johnson
+   orcid: 0000-0002-3600-7218
+   affiliation: 1
+ - name: Fabian Klötzl
+   orcid: 0000-0002-6930-0592
+   affiliation: 5
+ - name: David Koslicki
+   orcid: 0000-0002-0640-954X
+   affiliation: 4
+ - name: Marisa Lim
+   orcid: 0000-0003-2097-8818
+   affiliation: 1
+ - name: Ricky Lim
+   orcid: 0000-0003-1313-7076
+   affiliation: 9
+ - name: Ivan Ogasawara
+   orcid: 0000-0001-5049-4289
+   affiliation: 9
+ - name: Taylor Reiter
+   orcid: 0000-0002-7388-421X
+   affiliation: 1
+ - name: Camille Scott
+   orcid: 0000-0001-8822-8779
+   affiliation: 1
+ - name: Andreas Sjödin
+   orcid: 0000-0001-5350-4219
+   affiliation: 6
+ - name: Daniel Standage
+   orcid: 0000-0003-0342-8531
+   affiliation: 7
+ - name: S. Joshua Swamidass
+   orcid: 0000-0003-2191-0778
+   affiliation: 8
+ - name: Connor Tiffany
+   orcid: 0000-0001-8188-7720
+   affiliation: 9
+ - name: Pranathi Vemuri
+   orcid: 0000-0002-5748-9594
+   affiliation: 3
+ - name: Erik Young
+   orcid: 0000-0002-9195-9801
+   affiliation: 1
+ - name: C. Titus Brown
+   orcid: 0000-0001-6001-2677
+   corresponding: true
+   affiliation: 1
+affiliations:
+ - name: University of California, Davis
+   index: 1
+ - name: Woods Hole Oceanographic Institute
```

Comment:
Lokos good-- but should be Institution

Reply:
😂 I should have caught this - happens with Scripps all the time. Will fix!